src/App.js

```diff
@@ -35,7 +35,11 @@ const theme = {
   midtone: '#3C4271',
   secondary: '#4CD2B2',
   secondaryAlt: '#FF8541',
-
+  darkFuzzy: 'rgba(0, 0, 0, 0.15)',
+  lightGray: '#EEEEEE',
```

Comment:
Naming wise, I wouldn't call these colors by what they currently ARE. I'd name them instead based on what they're for. If we swap these colors out as part of a theme change, these names will no longer make sense.